packages/dso-toolkit/components/Componenten/_dialog/dialog.njk

```diff
@@ -1,16 +1,30 @@
-<div {{ className('dso-modal', [dialog.confirm, 'dso-confirm']) }} tabindex="-1" role="dialog" aria-labelledby="{{ dialog.id }}">
+<div {{ className('dso-modal', [dialog.confirm, 'dso-confirm']) }} tabindex="-1"
+  {{- attributes(
+    [dialog, 'role', 'dialog'],
+    [progress, 'role', 'alert'],
+    [dialog, 'aria-labelledby', dialog.id]
+  ) }}
+>
   <div class="dso-dialog" role="document">
-    <div class="dso-header">
-      <h2 id="{{ dialog.id }}">{{ dialog.heading }}</h2>
-      <button type="button" class="dso-close" aria-label="Sluiten"></button>
-    </div>
+    {% if dialog %}
+      <div class="dso-header">
+        <h2 id="{{ dialog.id }}">{{ dialog.heading }}</h2>
+        <button type="button" class="dso-close" aria-label="Sluiten"></button>
+      </div>
+    {% endif %}
     <div class="dso-body">
-      <p>{{ body }}</p>
-    </div>
-    <div class="dso-footer">
-      {% for button in dialog.buttons %}
-        {% render '@button', button %}
-      {% endfor %}
+      {% if dialog %}
+        <p>{{ body }}</p>
+      {% else %}
+        {% render '@progress-indicator-css', { size: 'small', block: true, labelid: 'prg-in-dialog', label: progress.label } %}
```

Comment:
Hier plaatste ik eerder de Web component, maar dat zorgde ervoor dat in fractal we zowel een Web component preview als een Markup component preview kregen, en daar wilden we zo graag vanaf.

Reply:
Yep 't is `aria-labelledby="{{ labelid }}"` -> de casing hebben we blijkbaar destijds over het hoofd gezien.